dashboard/test/ui/features/star_labs/ailab.feature

```diff
@@ -0,0 +1,56 @@
+Feature: AI Lab
+
+  @as_student
+  Scenario: Training a model with an in-house dataset
+    Given I am on "http://studio.code.org/s/allthethings/lessons/43/levels/1"
+    And I wait to see the dynamic instructions contain "selectDataset" key
+    And I close the instructions overlay if it exists
+    And I click selector ".ailab-image-hover"
+    And I wait to see the dynamic instructions contain "selectedDataset" key
+    And I see "#data-card"
+    And I click selector "#continue-button" 
+    And I wait to see the dynamic instructions contain "dataDisplayLabel" key
+    And I close the instructions overlay if it exists
+    When I select table column "0"
+    And I see "#column-inspector"
+    And I click selector "#select-label-button"
+    And I see "#remove-statement-label"
+    And I click selector "#continue-button"
+    And I wait to see the dynamic instructions contain "dataDisplayFeatures" key
+    And I close the instructions overlay if it exists
+    When I select table column "1"
+    And I wait for 3 seconds
+    And I see "#column-inspector"
+    And I click selector "#select-feature-button"
+    And I see "#remove-statement-feature"
+    And I click selector "#continue-button"
+    And I wait to see the dynamic instructions contain "trainModel" key
+    And I close the instructions overlay if it exists
+    And I see ".ailab-image-hover"
+    And I click selector "#continue-button"
+    And I wait to see the dynamic instructions contain "generateResults" key
+    And I see ".ailab-image-hover"
+    And I click selector "#continue-button"
+    And I wait to see the dynamic instructions contain "results" key
+    And I close the instructions overlay if it exists
+    And I see "#results"
+    And I wait for 1 seconds
```

Comment:
I know we wait for 1 second to show this, but maybe bumping to 2 seconds will give a bit of buffer if the browser lags...

Reply:
Sounds good!